Source/Common/utils.cpp

```diff
@@ -76,6 +76,47 @@ bool StringToUint(String const& s, uint64_t& v, int32_t base)
     return StringToUint4(s.data(), s.data() + s.size(), v, base);
 }
 
+HRESULT RunAsync(
+    AsyncWork&& work,
+    XTaskQueueHandle queue,
+    uint64_t delayInMs
+)
+{
+    auto workPointer = new AsyncWork(std::move(work));
+
```

Comment:
cleanup somewhere?

Reply:
ah, its done in  
   delete context;

---
In reply to: [294009762](https://github.com/microsoft/libHttpClient/pull/434#discussion_r294009762) [](ancestors = 294009762)